contrib/mpi-proxy-split/Makefile

```diff
@@ -95,23 +95,31 @@ check-unit: libmana.so
 ${DMTCP_ROOT}/lib/dmtcp/libmana.so: libmana.so
 	cp -f $< $@
 ${DMTCP_ROOT}/lib/dmtcp/libmpidummy.so:
-	make -C ${WRAPPERS_SRCDIR} install
+	make -C ${WRAPPERS_SRCDIR} mana 
 ${DMTCP_ROOT}/bin/lh_proxy:
-	make -C ${LOWER_HALF_SRCDIR} install
+	make -C ${LOWER_HALF_SRCDIR} mana
 ${DMTCP_ROOT}/bin/gethostbyname_proxy:
-	make -C ${LOWER_HALF_SRCDIR}/gethostbyname-static install
+	make -C ${LOWER_HALF_SRCDIR}/gethostbyname-static mana
 
 # ${WRAPPERS_SRCDIR}/libmpiwrappers.a ia a prerequisite for libmana.so,
 #   which is a prerequisite for ${DMTCP_ROOT}/lib/dmtcp/libmana.so
-# Always do 'make default' or 'make install' -- not 'make libmana.so'
+# Always do 'make default' or 'make mana' -- not 'make libmana.so'
 # MANA_COORD_OBJS needed for 'make mana' at top level.
-install: ${MANA_COORD_OBJS}
+mana: ${MANA_COORD_OBJS}
 	make -C ${WRAPPERS_SRCDIR} libmpiwrappers.a
 	make ${DMTCP_ROOT}/lib/dmtcp/libmana.so
 	make ${DMTCP_ROOT}/lib/dmtcp/libmpidummy.so
 	make ${DMTCP_ROOT}/bin/lh_proxy
 	make ${DMTCP_ROOT}/bin/gethostbyname_proxy
 
+install:
+	cp ${DMTCP_ROOT}/bin/lh_proxy ${DESTDIR}
+	cp ${DMTCP_ROOT}/bin/gethostbyname_proxy ${DESTDIR}
```

Comment:
We usually use `install` instead of `cp`, take a look at other DMTCP makefiles.

And only these two? I think MANA has other utils, like `mana_coordinator`

Reply:
oh i see what you mean. i'll see what i can do about it.